expo-zustand-styled-components/src/components/LinkButton/LinkButton.tsx

```diff
@@ -23,6 +23,8 @@ const LinkButton = ({ to, children, hasLine = false, onClick, ...rest }: LinkBut
     // Otherwise React Native for Web omits the `onClick` prop that's passed
     // You'll also need to pass `onPress` as `onClick` to the `View`
     // You can add hover effects using `onMouseEnter` and `onMouseLeave`
+    const { style, ...wrest } = rest;
```

Comment:
you can remove this and in the props destructured
```
const LinkButton = ({ to, children, hasLine = false, onClick, style, ...rest }: LinkButtonProps) => {

``` 

So that rest remains as rest.

Reply:
We have two logical components in this file, I don't want to add style={style} for the mobile which has the ...rest(including the style).